formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/DynamoUpdateService.scala

```diff
@@ -0,0 +1,81 @@
+package com.gu.identity.formstackbatonrequests
+
+import java.time.{Instant, LocalDate, LocalDateTime}
+
+import com.gu.identity.formstackbatonrequests.aws.{DynamoClient, SubmissionTableUpdateDate}
+import com.gu.identity.formstackbatonrequests.circeCodecs.{Form, FormSubmission}
+import com.typesafe.scalalogging.LazyLogging
+
+case class DynamoUpdateService(
+  formstackClient: FormstackRequestService,
+  dynamoClient: DynamoClient,
+  config: PerformLambdaConfig) extends LazyLogging {
+
+  def submissionsWithEmailAndAccount(submissions: List[FormSubmission], accountNumber: Int): List[SubmissionIdEmail] = {
+    val emailReg = """(?i)\b[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}\b""".r
```

Comment:
didn't we change this to 6 chars for the chars after the dot?

Reply:
Oh no, maybe I lost the change.. Thanks for catching this. I'll make sure nothing else was lost from that PR